git-tar/function/handler.go

```diff
@@ -266,14 +271,26 @@ func reportStatus(status *sdk.Status) {
 // available via the CDN. Note: given that the CDN has a 5-minute timeout - this optimization
 // may have the undesired effect of preventing a user from deploying within a 5 minute window
 // of renaming an incorrect "function.yml" to "stack.yml"
-func findStackFile(repo *sdk.PushEventRepository) (bool, error) {
+func findStackFile(pushEvent sdk.PushEvent) (bool, error) {
```

Comment:
How has this been unit tested? I'd like to see unit tests for this as part of the PR.